docs/source/_redirects

```diff
@@ -1,9 +1,13 @@
-/subgraphs/ /docs/federation/building-supergraphs/subgraphs-apollo-server/
+/subgraphs/ /docs/apollo-server/using-federation/apollo-subgraph-setup
 /gateway/ /docs/federation/building-supergraphs/router/
 /router/ /docs/federation/building-supergraphs/router/
 /supported-subgraphs/ /docs/federation/building-supergraphs/supported-subgraphs/
 /other-servers/ /docs/federation/building-supergraphs/supported-subgraphs/
 /federation-spec/ /docs/federation/subgraph-spec/
+/federation/building-supergraphs/router#nodejs-gateway-setup /docs/apollo-server/using-federation/apollo-gateway-setup
```

Comment:
Is section-specific (`#`) redirection actually a thing?

Reply:
You know, I'm not sure— I'll ask!

Reply:
Hm, I bet it isn't, and percentage-wise very few links would make use of this redirect anyway. I think this might just end up being a temporary rough edge for folks seeking this article, and we should omit this.